WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -281,9 +281,6 @@ def acknowledge_cookies
 
   private def user_params
     params.require(:user).permit(current_user.editable_fields_of_user(user_to_edit).to_a).tap do |user_params|
-      if user_params.key?(:delegate_status) && !User.delegate_status_requires_senior_delegate(user_params[:delegate_status])
-        user_params["region_id"] = nil
-      end
```

Comment:
Earlier, the senior_delegate_id was set to nil if a user didn't need senior delegate. But currently, this flow doesn't exist. We do change delegate roles through different API, so this code should be irrelevant now.